src/features/hourlies/components/HourlyReadingsDisplay.tsx

```diff
@@ -0,0 +1,148 @@
+            Hourly readings for weather station: {`${station.name} (${station.code})`}
+          </Typography>
+          <TableContainer>
+            <Table className={classes.table} size="small" aria-label="weather data table">
+              <TableBody>
+                <TableRow>
+                  <TableCell align="left">Date (PDT)</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {moment(v.datetime)
+                        .utcOffset(PDT_UTC_OFFSET)
+                        .format('YYYY-MM-DD HH:mm')}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Temperature</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.temperature.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Relative Humidity</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.relative_humidity.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Wind Speed</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.wind_speed.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Wind Direction</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.wind_direction.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Barometric Pressure</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.barometric_pressure.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Precipitation</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.precipitation.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>FFMC</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.ffmc?.toFixed(HOURLY_VALUES_DECIMAL)}
```

Comment:
Come to think of it, it might be better to display something like `-` instead of empty space, thought?

Reply:
I don't really know how prevalent this, or what the correct thing to show is. There's actually a lot more behind it - it's not calculating the value because there isn't a barometric reading (and error which is contained in the data coming from wf1). I don't want to assume that "-" or "n/a" is the correct thing to show here.